src/components/material-grid-related/MaterialGridRelated.tsx

```diff
@@ -19,26 +39,123 @@ const MaterialGridRelated: React.FC<MaterialGridRelatedProps> = ({ work }) => {
   const title = t("materialGridRelatedTitleText");
 
   const pid = getWorkPid(work);
-  const { data } = useWorkRecommendationsQuery({
-    pid: pid,
-    limit: 8
-  });
 
-  if (!data?.recommend.result) {
+  const { creators } = work.manifestations.bestRepresentation;
+  const { series } = work;
+  const seriesObject = first(series);
+  const flattenedCreators = flattenCreators(creators);
+  const creatorCqlString = prepareCreatorCql(flattenedCreators);
+
+  const { data: recommendationData, isLoading: recommendationLoading } =
+    useWorkRecommendationsQuery({
+      pid,
+      limit: 8
+    });
+
+  const { data: creatorData, isLoading: creatorLoading } =
+    useComplexSearchWithPaginationQuery({
+      cql: creatorCqlString,
+      limit: 8,
```

Comment:
This effectively limits the number of retrieved items to a maximum of 8 - which is the same number as the material grid will display without expanding. This is convenient but I wonder if it is as intended by the business.

Another approach would be to retrieve *all* the corresponding work (ids) and then let the component expand if requested by the user. This would make it possible to present extensive authorship or long running series.

This would likely also hand warranted a new query where we only retrieve a hitcount and work ids.

Oh well. This works.